release-notes/release-notes5102.md

```diff
@@ -0,0 +1,124 @@
+
+- `ephox-spelling.war`: 2.118.2
+- `ephox-hyperlinking.war`: 2.105.7
+- `ephox-image-proxy.war`: 2.105.6
+
+This version requires Java 8 or higher. For information on the removal of Java 7 support, see: [Removal of Java 7 support for TinyMCE 5.3 and later]({{site.baseurl}}/release-notes/release-notes53/#removalofjava7support).
+
+For information on:
+
+- The Spell Checker Pro plugin, see: [Spell Checker Pro plugin]({{site.baseurl}}/plugins/premium/tinymcespellchecker/).
+- The Link Checker plugin, see: [Link Checker plugin]({{site.baseurl}}/plugins/premium/linkchecker/).
+- The Image Tools plugin, see: [Image Tools plugin]({{site.baseurl}}/plugins/opensource/imagetools/).
+- The Enhanced Media Embed plugin, see: [Enhanced Media Embed plugin]({{site.baseurl}}/plugins/premium/mediaembed/).
+- Deploying the server-side components, see: [Server-side component installation]({{site.baseurl}}/enterprise/server/).
+
+### Security related dependency update for all 3 server-side services
+
+All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ severity vulnerability fixes.
+
+### Updating the self-hosted server-side components
+
+The new versions of the server-side services provide updates for the Java-based server-side components. To deploy the updated version of the server-side components:
+
+1. Update your Java Application Server to the minimum required version:
+
+    - Eclipse Jetty 9.4 or later
+    - Apache Tomcat:
+        - 9 or later
+        - 8.5.12+
+        - 8.0.42+
+        - 7.0.76+
+
+2. Replace the existing server-side `.war` files with the `.war` files bundled with {{site.productname}} 5.10.2 or later.
+
+For information on:
+
+- Deploying the server-side components, see: [Server-side component installation]({{site.baseurl}}/enterprise/server/).
+- Deploying the server-side components using Docker, see: [Containerized service deployments]({{site.baseurl}}/enterprise/server/dockerservices/).
+
+## General bug fixes
+
+{{site.productname}} 5.10.2 provides fixes for the following bugs:
+
+- Internal selectors were appearing in the style list when using the `importcss` plugin.
+
+## Security fixes
+
+{{site.productname}} 5.10.2 provides fixes for the following security issues:
+
+All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ severity vulnerability fixes.
+
+For information on the server-side components updates, see: [Accompanying Premium self-hosted server-side component changes](#accompanyingpremiumself-hostedserver-sidecomponentchanges).
+
+## Known issues
+
+This section describes issues that users of {{site.productname}} 5.10.2 may encounter and possible workarounds for these issues.
+
+- [](#)
+
+###
+
+Issue
+: This issue affects ....
+
+Workaround
+: There was no known workaround at the time of the release.
+
```

Comment:
```suggestion
```
Just putting this here so we don't forget as I would hope we have nothing to add here.